crates/schema-impl/src/lib.rs

```diff
@@ -221,9 +200,9 @@ impl Schemas {
     pub fn apply_updates(
         &self,
         updates: impl IntoIterator<Item = SchemasUpdateCommand>,
-    ) -> Result<(), SchemasUpdateError> {
+    ) -> Result<(), Error> {
```

Comment:
Why is this method returning an `Error`?

Reply:
Sorry this is a leftover, i removed it.